test/unit/vl-alert.test.html

```diff
@@ -38,7 +46,7 @@
         const alert = fixture('vl-alert-fixture');
         const value = 'title';
         alert.setAttribute('title', value);
-        assert.isTrue(alert._titleElement.textContent == value);
+        assert.isTrue(alert._titleSlotElement.textContent == value);
```

Comment:
miss vergelijken met ===

Reply:
Nog beter is het gebruik van de assert.equal. PR is al gemerged, dus stel ik voor om het voorlopig zo te laten aangezien het niet verkeerd is.